examples/run_examples.py

```diff
@@ -55,13 +56,19 @@
 req_version['saga_batch.py'] = (2019,3)
 req_version['dbscan_batch.py'] = (2019,5)
 req_version['lasso_regression_batch.py'] = (2019,5)
-req_version['elastic_net_batch.py'] = (2021,5)
+req_version['elastic_net_batch.py'] = (2022,0) # (2021,5) is relevant but there is an issue with versions number
+req_version['sycl/bf_knn_classification_batch.py'] = (2022,0) # there is an issue with versions number
+req_version['sycl/gradient_boosted_regression_batch.py'] = (2022,0) # there is an issue with versions number
```

Comment:
what's the issue with the version numbers?

Reply:
Our checks of versions suppose that algorithm or some behavior is available in DAAL of some version and above. But now this rule is not working properly. Another reason is that it is possible to get versions like (2021,103). We decided to discuss and rework this separately.

Reply:
We've discuss versions briefly. 
1. New features are introduced in both version and for example it can be enabled in 2020.1 and in 2021.Beta Update 5. so check logic should be updated.
2. It's interesting that we see 2021.105 versioning as it supposed to be 2021.1.0.5 and most importantly now we are at beta stage and this should be reflected in logic as  this is is true condition 2021.0 > 2021.105B

Reply:
This have been fixed